src/.nuget/Microsoft.NETCore.ILAsm/runtime.Android.Microsoft.NETCore.ILAsm.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Given that Android is sufficiently distinct, but still Linux based, I am fine if a new OS group for it is used while leveraging the existing Linux definitions of the components.

CC @chcosta